### ISSUE TITLE
Team name and website does not appear in Firefox

### ISSUE THREAD

**Issue by nathanmutin** [frc1418/FRCNewTab#42]
First, I want to tell you that I love your work!
However, I use Firefox 62.0.2 and the team names and website doesn't appear even if I check the option.
![Capture](https://user-images.githubusercontent.com/32647745/46256693-7caf0a80-c4ae-11e8-8c35-2b79bb4de7cc.png)

**Comment by ErikBoesen** [comment]
I can reproduce; looks like I forgot to initialize the `localStorage.dynamicTitle` variable. I will push a fix as fast as possible.

**Comment by ErikBoesen** [comment]
Nope, I'm an idiot and totally forgot what `dynamicTitle` is. The problem is that I was inserting an `<a>` element for a link to the team website into the DOM without actually putting the text into it. Should be fixed momentarily.

**Comment by nathanmutin** [comment]
The issue is resolved. Thanks again.
